### PR TITLE
fix: improve settings page and overall

### LINKED ISSUE #1135: Add "or click the 'new item' button to create more elements" to the dropzone helper

### PR DESCRIPTION
**Pull request by spaenleh**
Added the "New Item" button option on the drop zone helper: 
add new item in dropzone

Home page with transparent table:
table on purple

New settings:
new settings
new settings 2

- [ ] link settings need to be updated and redesigned

fix #1135

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@graasp/query-client": "3.5.0",
     "@graasp/sdk": "4.7.6",
     "@graasp/translations": "1.27.0",
-    "@graasp/ui": "4.15.1",
+    "@graasp/ui": "github:graasp/graasp-ui#update-table-for-background",
```
Comment (pyphilia) [inline package.json:27]: To update.

File: src/components/item/publish/CustomizedTagsEdit.tsx [hunk 2]
```diff
@@ -60,13 +53,21 @@ const CustomizedTagsEdit = ({ item, disabled }: Props): JSX.Element => {
...
-      </Typography>
-      <Typography variant="body1">
-        {translateBuilder(BUILDER.ITEM_TAGS_INFORMATION)}
-      </Typography>
+    <Stack
+      gap={1}
+      // bgcolor="white"
+      // p={2}
+      // borderRadius={2}
+      // border="1px solid #ddd"
```
Comment (pyphilia) [inline src/components/item/publish/CustomizedTagsEdit.tsx:61]: Remove?

File: src/components/item/settings/ClearChatButton.tsx [hunk 3]
```diff
@@ -28,69 +27,38 @@ import {
...
-          </Tooltip>
-        );
-      case ButtonVariants.Button:
-      default:
-        return (
+  return (
+    <>
+      <Stack direction="column" spacing={2}>
+        <Typography variant="body1" maxWidth="50ch">
+          {}
```
Comment (pyphilia) [inline src/components/item/settings/ClearChatButton.tsx:49]: ?

File: src/components/item/settings/GeolocationPicker.tsx [hunk 4]
```diff
@@ -50,40 +50,40 @@ const GeolocationPicker = ({
...
-          <Typography variant="caption">
-            {t(BUILDER.ITEM_SETTINGS_GEOLOCATION_INHERITED_EXPLANATION)}
-          </Typography>
-        )}
+    <Stack
+      gap={1}
+      // bgcolor="white"
+      // p={2}
+      // borderRadius={2}
+      // border="1px solid #ddd"
```
Comment (pyphilia) [inline src/components/item/settings/GeolocationPicker.tsx:58]: Remove?

File: src/components/item/settings/ItemLicenseSettings.tsx [hunk 5]
```diff
@@ -21,10 +20,10 @@ const ItemLicenseSettings = (): JSX.Element => {
...
   const { item } = useOutletContext<OutletType>();
 
-  const handleClick = () => {
-    const url = CC_LICENSE_ABOUT_URL;
-    redirect(window, url, { openInNewTab: true });
-  };
+  // const handleClick = () => {
+  //   const url = CC_LICENSE_ABOUT_URL;
+  //   redirect(window, url, { openInNewTab: true });
+  // };
```
Comment (pyphilia) [inline src/components/item/settings/ItemLicenseSettings.tsx:26]: Remove?

File: src/components/item/settings/ItemMetadataContent.tsx [hunk 6]
```diff
@@ -69,10 +70,10 @@ const ItemMetadataContent = (): JSX.Element => {
   };
 
   return (
-    <>
-      <ThumbnailSetting item={item} />
-
-      <TableContainer sx={{ p: 2, boxSizing: 'border-box' }}>
+    <Box
+    // bgcolor="white" borderRadius={2} border="1px solid #ddd"
```
Comment (pyphilia) [inline src/components/item/settings/ItemMetadataContent.tsx:74]: Remove?

File: src/langs/fr.json [hunk 7]
```diff
@@ -320,7 +319,7 @@
   "ITEM_SETTINGS_GEOLOCATION_PLACEHOLDER": "Entrer l'adresse ici",
   "ITEM_SETTINGS_GEOLOCATION_NO_ADDRESS": "Aucune adresse trouvée",
   "ITEM_SETTINGS_GEOLOCATION_TITLE": "Localisation Géographique",
-  "ITEM_SETTINGS_CLEAR_GEOLOCATION": "Supprimer la Localisation Géolocalisation",
+  "ITEM_SETTINGS_GEOLOCATION_CLEAR": "Supprimer la Localisation Géolocalisation",
```
Comment (pyphilia) [inline src/langs/fr.json:322]: My bad.
```suggestion
  "ITEM_SETTINGS_GEOLOCATION_CLEAR": "Supprimer la Géolocalisation",
```

**Review by pyphilia — CHANGES_REQUESTED** [review]
I'm not fan of this "3 options" empty content. What about removing the browse files with new item? For me it makes no sense to have more than one button here. Or use this layout:
image
image

Thanks for all the changes!!! 🍇

**Review by ReidyT — COMMENTED** [review]
Thanks for all these improvements, it looks nice !
 
I agree with Kim concerning the three options to add content on empty folder.
Also, I think it is a little weird the size of the language row comparing to the rest of the table. Maybe it could be nicer to have smaller `select` ? I have another comment about the select of the language who is not in the center of the row...

Except that, LGTM 👍

**Comment by spaenleh** [comment]
@ReidyT see https://github.com/graasp/graasp-ui/pull/825 for the centering of the select :wink:

**Comment by spaenleh** [comment]
@pyphilia @ReidyT How about this layout ?
Screenshot 2024-04-25 at 22 36 47

File: package.json [hunk 8]
```diff
@@ -24,7 +24,7 @@
     "@graasp/query-client": "3.5.0",
     "@graasp/sdk": "4.7.6",
     "@graasp/translations": "1.27.0",
-    "@graasp/ui": "4.15.1",
+    "@graasp/ui": "github:graasp/graasp-ui#remove-margin-when-no-label",
```
Comment (ReidyT) [inline package.json:27]: Just a reminder 😄

**Review by ReidyT — APPROVED** [review]
Yeah ! I really like the new empty box 👌

**Review by pyphilia — APPROVED** [review]

**Pull request merged**